run aio/develop/run-npm-on-container.sh failed

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!-->

##### Environment
<!-- Describe your setup. Versions of Node.js, Go etc. are needed only from developers. -->

```
Dashboard version: Master
Operating system: MacOs 10.13.6
```

##### Steps to reproduce
<!-- Describe all steps needed to reproduce the issue. It is a good place to use numbered list. -->
```bash
git clone  https://github.com/kubernetes/dashboard.git
cd dashboard/aio/develop
./run-npm-on-container.sh
```

##### Observed result
```bash
Hit:1 http://cdn-fastly.deb.debian.org/debian buster InRelease
Hit:3 https://deb.nodesource.com/node_11.x buster InRelease
Hit:2 http://security-cdn.debian.org/debian-security buster/updates InRelease
Hit:4 http://cdn-fastly.deb.debian.org/debian buster-updates InRelease
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package openjdk-8-jre
The command '/bin/sh -c curl -sL https://deb.nodesource.com/setup_11.x | bash -   && apt-get update   && apt-get install -y --no-install-recommends 	openjdk-8-jre 	nodejs 	patch 	chromium 	bc sudo 	gosu 	nano 	less 	&& rm -rf /var/lib/apt/lists/* 	&& apt-get clean' returned a non-zero code: 100
Run container for development
```

##### Comments
<!-- If you have any comments or more details, put them here. -->
I think it's Dockerfile problem.
I run a container of `golang:1.12` ,then run `apt update && apt install openjdk-8-jre`. The terminal shows the same error.
Is this a problem with my personal environment?

I have a similar problem.

In my case, I had the following steps.

I had minikube on mac.

```shell
$ export K8S_DASHBOARD_KUBECONFIG=/Users/smartkuk/.kube/config
```

```shell
$ export GO111MODULE=on
```

```shell
$ aio/develop/run-npm-on-container.sh
```

I face this problem.

```shell
Remove existing container k8s-dashboard-dev
Error: No such container: k8s-dashboard-dev
Start building container image for development
Sending build context to Docker daemon  36.33MB
Step 1/20 : FROM golang:1.12
 ---> be63d15101cb
Step 2/20 : RUN curl -sL https://deb.nodesource.com/setup_11.x | bash -   && apt-get install -y --no-install-recommends 	openjdk-8-jre 	nodejs 	patch 	chromium 	bc 	sudo 	gosu 	nano 	less 	&& rm -rf /var/lib/apt/lists/* 	&& apt-get clean
 ---> Running in 3a2d76d94985

## Installing the NodeSource Node.js 11.x repo...


## Populating apt-get cache...

+ apt-get update
Get:1 http://security.debian.org/debian-security buster/updates InRelease [39.1 kB]
Get:2 http://deb.debian.org/debian buster InRelease [118 kB]
Get:3 http://security.debian.org/debian-security buster/updates/main amd64 Packages [49.4 kB]
Get:4 http://deb.debian.org/debian buster-updates InRelease [46.8 kB]
Get:5 http://deb.debian.org/debian buster/main amd64 Packages [7897 kB]
Fetched 8150 kB in 2s (3615 kB/s)
Reading package lists...

## Installing packages required for setup: lsb-release...

+ apt-get install -y lsb-release > /dev/null 2>&1

## Confirming "buster" is supported...

+ curl -sLf -o /dev/null 'https://deb.nodesource.com/node_11.x/dists/buster/Release'

## Adding the NodeSource signing key to your keyring...

+ curl -s https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add -
Warning: apt-key output should not be parsed (stdout is not a terminal)
OK

## Creating apt sources list file for the NodeSource Node.js 11.x repo...

+ echo 'deb https://deb.nodesource.com/node_11.x buster main' > /etc/apt/sources.list.d/nodesource.list
+ echo 'deb-src https://deb.nodesource.com/node_11.x buster main' >> /etc/apt/sources.list.d/nodesource.list

## Running `apt-get update` for you...

+ apt-get update
Hit:1 http://deb.debian.org/debian buster InRelease
Hit:2 http://security.debian.org/debian-security buster/updates InRelease
Hit:3 http://deb.debian.org/debian buster-updates InRelease
Get:4 https://deb.nodesource.com/node_11.x buster InRelease [4584 B]
Get:5 https://deb.nodesource.com/node_11.x buster/main amd64 Packages [768 B]
Fetched 5352 B in 1s (6146 B/s)
Reading package lists...

## Run `sudo apt-get install -y nodejs` to install Node.js 11.x and npm
## You may also need development tools to build native addons:
     sudo apt-get install gcc g++ make
## To install the Yarn package manager, run:
     curl -sL https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
     echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
     sudo apt-get update && sudo apt-get install yarn


Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package openjdk-8-jre
The command '/bin/sh -c curl -sL https://deb.nodesource.com/setup_11.x | bash -   && apt-get install -y --no-install-recommends 	openjdk-8-jre 	nodejs 	patch 	chromium 	bc 	sudo 	gosu 	nano 	less 	&& rm -rf /var/lib/apt/lists/* 	&& apt-get clean' returned a non-zero code: 100
Run container for development
2019/07/23 01:05:11 Starting overwatch
2019/07/23 01:05:11 Using namespace: kube-system
2019/07/23 01:05:11 Using in-cluster config to connect to apiserver
2019/07/23 01:05:11 Could not init in cluster config: unable to load in-cluster configuration, KUBERNETES_SERVICE_HOST and KUBERNETES_SERVICE_PORT must be defined
panic: could not create client config

goroutine 1 [running]:
github.com/kubernetes/dashboard/src/app/backend/client.(*clientManager).initInsecureConfig(0xc00023e000)
	/Users/smartkuk/Documents/git_clones/github/dashboard/src/app/backend/client/manager.go:508 +0x16a
github.com/kubernetes/dashboard/src/app/backend/client.(*clientManager).initInsecureClients(0xc00023e000)
	/Users/smartkuk/Documents/git_clones/github/dashboard/src/app/backend/client/manager.go:484 +0x2f
github.com/kubernetes/dashboard/src/app/backend/client.(*clientManager).init(0xc00023e000)
	/Users/smartkuk/Documents/git_clones/github/dashboard/src/app/backend/client/manager.go:446 +0x39
github.com/kubernetes/dashboard/src/app/backend/client.NewClientManager(...)
	/Users/smartkuk/Documents/git_clones/github/dashboard/src/app/backend/client/manager.go:528
main.main()
	/Users/smartkuk/Documents/git_clones/github/dashboard/src/app/backend/dashboard.go:105 +0x213
```

--- a/aio/develop/Dockerfile
+++ b/aio/develop/Dockerfile
@@ -19,7 +19,7 @@
 
 # golang is based on debian:jessie
 # Specify version to clarify which version we use.
-FROM golang:1.12
+FROM golang:1.10
 
 # Install Node.js. Go is already installed.
 # A small tweak, apt-get update is already run by the nodejs setup script,

--- a/aio/develop/Dockerfile
+++ b/aio/develop/Dockerfile
@@ -19,14 +19,13 @@
 
 # golang is based on debian:jessie
 # Specify version to clarify which version we use.
-FROM golang:1.10
+FROM golang:1.12
 
 # Install Node.js. Go is already installed.
 # A small tweak, apt-get update is already run by the nodejs setup script,
 # so there's no need to run it again.
 RUN curl -sL https://deb.nodesource.com/setup_11.x | bash - \
   && apt-get install -y --no-install-recommends \
-	openjdk-8-jre \
 	nodejs \
 	patch \
 	chromium \

